src/Website/presence.json

```
@@ -39,6 +39,10 @@
 		"message": "Supported URLs",
 		"description": "One of the strings from the Information section. [on webite -> Supported URLs: (array)]"
 	},
+	"presence.sections.information.moreurls": {
+		"message": "This Presence supports more URLs than listed below. Hover for more information.",
```

If @Bas950 and @TheDropX agree
```suggestion
		"message": "This Presence supports more URLs than listed below through the use of RegExp. Hover to see the pattern",
```

---

I exist too 🥲

---

Sorry I thought you weren’t looking at the Website atm 😳

---

LGTM

---

lgtm